src/QsCompiler/Compiler/PluginInterface.cs

```diff
@@ -17,6 +17,12 @@ namespace Microsoft.Quantum.QsCompiler
     /// </summary>
     public static class RewriteStepPriorities
     {
+        /// <summary>
+        /// Priority of the built-in transformation that removes all
+        /// unused callables from the syntax tree.
+        /// </summary>
+        public const int SyntaxTreeTrimming = 1200;
```

Comment:
Minor:
Are there any limitations for this value? Must this value be above or below (or both) certain threshold?

Reply:
There are no limitations that I know about. I think that the exact numbers are somewhat arbitrary except the ordering they establish (which is not arbitrary).

Reply:
OK

Reply:
There are indeed no limitations aside from data type limitations; the idea is just to leave enough space in between for other transformations that are added (by us or by compiler extensions).